### ISSUE TITLE
Problem with Setup the Plugin

### ISSUE THREAD

**Issue by Hanimaniac** [spatialdude/homebridge-ecowitt#31]
Hello Everyone,

I can't get the Plugin to work with my Homebridge.
I like to use the Soil Moisture Sensor in my Homekit.

```
Show the Homebridge logs here, remove any sensitive information.
```
[14/05/2024, 21:43:14] [Ecowitt] Creating data report service
[14/05/2024, 21:43:14] [Ecowitt]   Port: 8080
[14/05/2024, 21:43:14] [Ecowitt]   Path: /data/report/
[14/05/2024, 21:43:14] [Ecowitt] Unregistering cached accessories: 0
[14/05/2024, 21:43:14] [Ecowitt] Listening for data reports on: http://[::]:8080
[14/05/2024, 21:44:29] [Ecowitt] Data source address: ::ffff:1XX.XXX.XXX.31
[14/05/2024, 21:44:29] [Ecowitt] Request: POST /data/report/ HTTP/1.1
host: 1XX.XXX.XXX.157
connection: Close
[14/05/2024, 21:44:29] [Ecowitt] Not configured for data reports from this base station: {
  "PASSKEY": "E16FBFBBB01952F8EFB529E187AEAC82",
  "stationtype": "GW1100A_V2.3.2",
  "runtime": "172597",
  "heap": "25600",
  "dateutc": "2024-05-14 19:44:29",
  "tempinf": "77.18",
  "humidityin": "37",
  "baromrelin": "29.669",
  "baromabsin": "29.669",
  "soilmoisture1": "27",
  "soilad1": "173",
  "soilbatt1": "1.5",
  "freq": "868M",
  "model": "GW1100A",
  "interval": "16"
}

1XX.XXX.XXX.31 - IP Adress Ecowitt Gateway GW1100
1XX.XXX.XXX.157 - IP Adress Homebridge

**Plugin Config:**

{
    "mac": " 11:11:11:11:11:1D",
    "port": 8080,
    "path": "/data/report/",
    "ws": {
        "hide": true,
        "uv": {
            "hide": false,
            "threshold": 6
        },
        "solarradiation": {
            "hide": false,
            "luxFactor": 126.7
        },
        "wind": {
            "units": "beaufort",
            "speedThreshold": 1,
            "gustThreshold": 3,
            "maxDailyGustThreshold": 8
        },
        "rain": {
            "units": "mm"
        }
    },
    "thbin": {
        "hide": false
    },
    "th": {
        "hide": true
    },
    "soil": {
        "hide": false,
        "name1": "Bodenfeuchte"
    },
    "leak": {
        "hide": true
    },
    "pm25": {
        "hide": true
    },
    "lightning": {
        "hide": true,
        "units": "km"
    },
    "platform": "Ecowitt"
}

This is the customized Weather Service of the Ecowitt Gateway

<img width="672" alt="Bildschirmfoto 2024-05-14 um 21 56 44" src="https://github.com/spatialdude/homebridge-ecowitt/assets/45320732/96fcd1ef-35f7-491a-89a2-ef4c38ddb7ea">

* **Plugin Version: v1.1.3**:
* **Homebridge Version**: V1.8.1

**Comment by buckotronic** [comment]
@Hanimaniac hi, did you manage to run the plugin on homebridge?

**Comment by Hanimaniac** [comment]
No, unfortunately I couldn't get the plugin to run. It seems that the gateway GW1100A is not supported (although it is supported in the description). 

Until there is a solution, I am on the iot-soil-moisture homekit Sensor. 
https://github.com/shubhsheth/iot-moisture-sensor

**Comment by rhockenbury** [comment]
Hi @Hanimaniac & @buckotronic.  I have a new Ecowitt plugin available (v2), check out [Homebridge Ecowitt Weather Sensors](https://github.com/rhockenbury/homebridge-ecowitt-weather-sensors) which can be installed through the Homebridge UI. I've verified that this new plugin works with the data report that you provided above.  Would appreciate any feedback if you have time to give this new version a try, and [let me know if you hit any issues](https://github.com/rhockenbury/homebridge-ecowitt-weather-sensors/issues/new).
